libs/blocks/editorial-card/editorial-card.js

```diff
@@ -0,0 +1,128 @@
+import { createTag, loadStyle, getConfig } from '../../utils/utils.js';
+import { decorateBlockBg, decorateBlockText, decorateBlockHrs, decorateTextOverrides, applyHoverPlay } from '../../utils/decorate.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+async function loadIconography() {
+  try {
+    const stylePromise = new Promise((resolve) => {
+      loadStyle(`${base}/styles/iconography.css`, resolve);
+    });
+    await Promise.all([stylePromise]);
+  } catch (err) {
+    window.lana?.log(`Failed to load iconography module: ${err}`);
+  }
+}
+
+function decorateLockupFromContent(el) {
+  const rows = el.querySelectorAll(':scope > div > p');
+  const firstRowImg = rows[0]?.querySelector('img');
+  if (!firstRowImg) return;
+  loadIconography();
```

Comment:
To make sure the styles are properly loaded before continuing decoration you'd need to `await decorateForeground()` which will `await decorateLockupFromContent()` which will in turn `await loadIconography()` 

Currently you add the style sheet to the document, and they will start loading - but continue decoration. Hence you might get a flash of unstyled content // CLS.